CHANGES.rst

```diff
@@ -5,8 +5,7 @@ Enhancements
 ------------
 * Brings back support for legacy predictions (api/v1) and a new parameter for specifying api version (``--api_version``).
   Check ``batch_scoring --help`` for a list of valid options and the default value.
-
-
+* Default for timeout is now None, meaning that the code does not enforce a timeout for operations to the server. This allows completion of runs with higher numbers of threads, particularly in MacOS.
```

Comment:
Sorry, one extra: we should let them know that this value is still configurable.